website/views/shared/modals/achievements.jade

```diff
@@ -112,7 +112,7 @@ script(id='modals/achievements/contributor.html', type='text/ng-template')
       h3(style='margin-bottom:0')=env.t('modalContribAchievement')
       +achievementAvatar('boot',0)
       !=env.t('contribModal', {name: "{{user.profile.name}}", level: "{{user.contributor.level}}"}) + ' '
-      a(href='http://habitica.wikia.com/wiki/Contributor_Rewards' target='_blank')=env.t('contribLink')
+      a(href=env.t('conLearnLink') target='_blank')=env.t('contribLink')
```

Comment:
`conLearnLink` is the URL for the "Contributing to Habitica" page, not the "Contributor Rewards" page.